rakefile.rb

```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)

```

Comment:
@blairconrad I've amended so that we move the release notes forward in GitHub. How does it look now?